README.md

```diff
@@ -103,6 +104,11 @@ g stands for global installation. If you use -g later you can use the CLI in any
  1. Create update-password component .To do that ,we need to use the following command :ng g c update-password
  2. User change password is figure out by this tool.
 
+ ### For Changepassword
```

Comment:
@Bithika-Redapple Good documentation. Remember to 
1. Title case -> "Change Password" (drop the for)
2. Separate words -> "Change Password"